inst/config/config.yml

```diff
@@ -3,18 +3,65 @@ default:
     - simID
     - source
     - time
+  group:
+    - time
+  group_ds:  # used in gen_data_weighted_ds as well as gen_data_weighted_burden
+    - simID
+  summary_columns_rf:  # Variables that are used to calculate the summary statistics
```

Comment:
Suggested notes:
# Risk factor variables to calculate summary statistics, such as mean, min and max